spyder/plugins/editor/plugin.py

```diff
@@ -1007,18 +1013,12 @@ def _toggle_checkable_action(self, checked, editorstack_method, conf_name):
         """
         if self.editorstacks:
             for editorstack in self.editorstacks:
-                try:
-                    editorstack.__getattribute__(editorstack_method)(checked)
-                except AttributeError as e:
-                    logger.error(e, exc_info=True)

```

Comment:
Please restore these three lines above. They are needed for the other options in the Source menu to work (e.g. to activate/deactivate indent guides, etc).